admin/config.js

```diff
@@ -28,7 +28,8 @@ module.exports = {
     allowedWords: process.env.ALLOWED_WORDS || 'aaa,bcd,dcd,tfg,bxx',
     pinSubmissionMaxAttempts: process.env.PIN_SUBMISSION_MAX_ATTEMPTS || 100,
     helplineNumber: process.env.HELPLINE_NUMBER || '0345 278 8080',
-    pupilCensusMaxSizeFileUploadMb: process.env.PUPIL_CENSUS_MAX_FILE_UPLOAD_MB || 100 * 1024 * 1024
+    pupilCensusMaxSizeFileUploadMb: process.env.PUPIL_CENSUS_MAX_FILE_UPLOAD_MB || 100 * 1024 * 1024,
+    psychometricianReportMaxSizeFileUploadMb: process.env.PSYCHOMETRICIAN_REPORT_MAX_FILE_UPLOAD_MB || 100 * 1024 * 1024
```

Comment:
would you mind abbreviating this to `PS_REPORT_MAX_FILE_UPLOAD_MB` as i'm the poor sod who has to type these into the environments 😄